pkg/ctl/cmdutils/create_cluster.go

```diff
@@ -13,6 +13,7 @@ type CreateClusterCmdParams struct {
 	SetContext                  bool
 	AvailabilityZones           []string
 	InstallWindowsVPCController bool
+	InstallNeuronDevicePlugin   bool
```

Comment:
We should ask Kalbir if this should be a CLI flag or a field in the config file?